docs/rules/consistent-output.md

```diff
@@ -13,51 +13,45 @@ This rule aims to ensure that if any invalid test cases have output assertions,
 Examples of **incorrect** code for this rule:
 
 ```js
-/* eslint eslint-plugin/consistent-output: error */
-
```

Comment:
why removing the inline config?

Reply:
I just added the inline configs back. I thought it seemed redundant and not all of the rule doc examples use it. But we can leave as-is for now.